config/secrets.yml

```diff
@@ -1,5 +1,5 @@
 default: &default
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  secret_key_base: 8370c4c3531fe72f6988d1a90fc8aa0be2f38492340ac7ec78cf7875797e19bd788e28a1e28ffebb261ba958c387298b082b708c45e46922c1268b16ea5eff15
```

Comment:
I just put a random string hash cause i was struggling with getting the app up and runnning with docker, this should be ignored